packages/frontend/src/features/catalog/components/CatalogPanel.tsx

```diff
@@ -414,13 +433,61 @@ export const CatalogPanel: FC = () => {
                 </Group>
             </Group>
 
-            <CatalogTree
-                tree={catalogTree}
-                projectUuid={projectUuid}
-                searchString={debouncedSearch}
-                selection={selection}
-                onItemClick={selectAndGetMetadata}
-            />
+            {noResults ? (
+                <Paper
+                    p="xl"
+                    radius="lg"
+                    sx={(theme) => ({
+                        backgroundColor: theme.colors.gray[1],
+                        border: `1px solid ${theme.colors.gray[3]}`,
+                    })}
+                >
+                    {noTables ? (
+                        <SuboptimalState
+                            icon={IconTable}
+                            title="No tables found in this project"
+                            description={
+                                "Tables are the starting point to any data exploration in Lightdash. They come from dbt models that have been defined in your dbt project's .yml files."
+                            }
+                            action={
+                                <LinkButton
+                                    color="dark"
+                                    href="https://docs.lightdash.com/guides/adding-tables-to-lightdash"
```

Comment:
should we add a `target='_blank'` here ?

Reply:
Yea, good call. Done. I think that might fix the link out as well? It works locally.

Reply:
Styled the button here as well :https://github.com/lightdash/lightdash/pull/10264/commits/ce2436a5f3aad1bbfc7cb3b0574c41f7165d962e